app/uk/gov/hmrc/agentassurance/services/EntityCheckService.scala

```diff
@@ -140,9 +135,7 @@ class EntityCheckService @Inject() (
     def sendEmail(
       agentRecord: AgentDetailsDesResponse,
       entityCheckExceptions: Seq[EntityCheckException]
-    ): Future[Unit] = {
-      val formatter = DateTimeFormatter.ofPattern("d MMMM yyyy h:mma")
-
+    )(implicit clock: Clock): Future[Unit] = {
```

Comment:
This should be injected instead since the clock is global and never needs overriding